src/index.scss

```diff
@@ -29,9 +29,11 @@ body {
 
 .FloodsApp {
   margin: 0 auto;
-  height: calc(100% - 1px);
+  height: 100%; //previously calc(100% - 1px)
```

Comment:
definitely worth verifying this doesn't break mobile safari, I'm pretty sure this led to the page trying to scroll all the time and making it 1px smaller made it not try to scroll.

Reply:
Checked - mobile Safari works fine